presto-orc/src/main/java/com/facebook/presto/orc/reader/LongDictionarySelectiveStreamReader.java

```diff
@@ -278,13 +278,14 @@ private void openRowGroup()
         if (!dictionaryOpen && dictionarySize > 0) {
             if (dictionary.length < dictionarySize) {
                 dictionary = new long[dictionarySize];
+                systemMemoryContext.setBytes(sizeOf(dictionary));
```

Comment:
This change doesn't seem right and is not part of nulling member variables in close.